<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sign_up</class>
 <widget class="QWidget" name="sign_up">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>欢迎来到北京大学图书馆</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLineEdit" name="NewIDEdit">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>170</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordEdit_1">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>240</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordEdit_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>310</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="backgroundlabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>425</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>750</width>
     <height>425</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image :url(:/background/picture/zhuce3.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="titlelabel">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>40</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 16pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>Welcome to</string>
   </property>
  </widget>
  <widget class="QLabel" name="titlelabel_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>80</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 16pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>Peking University Library</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SureButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>350</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>Sure</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>350</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="newlabel_1">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>150</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>UserID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="newlabel_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>220</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;
</string>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="newlabel_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>290</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>Ensure your password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="titlelabel_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image :url(:/background/picture/tushuguantitle.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>backgroundlabel</zorder>
  <zorder>NewIDEdit</zorder>
  <zorder>PasswordEdit_1</zorder>
  <zorder>PasswordEdit_2</zorder>
  <zorder>titlelabel</zorder>
  <zorder>titlelabel_2</zorder>
  <zorder>SureButton</zorder>
  <zorder>clearButton</zorder>
  <zorder>newlabel_1</zorder>
  <zorder>newlabel_2</zorder>
  <zorder>newlabel_3</zorder>
  <zorder>titlelabel_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
